docs/tutorial/mint-nftstorage-polygon.md

```diff
@@ -176,6 +176,27 @@ Your output should look like the listing below, where `HASH` is the CID to the a
 Metadata stored on Filecoin/IPFS at URL: ipfs://HASH/metadata.json
 ```
 
+I wrote this to store all images from one directory into a folder 
```

Comment:
Could you write in a style consistent with the original content? (For instance, the pronoun is "we" instead of first-person "I").